ras-arm-handler.c

```diff
@@ -207,6 +206,9 @@ int ras_arm_event_handler(struct trace_seq *s,
 	ev.psci_state = val;
 	trace_seq_printf(s, "\n psci_state: %d", ev.psci_state);
 
+#ifdef HAVE_AMP_NS_DECODE
+	int len = 0;
+
```

Comment:
Coding style nit: please keep all variable declarations at the beginning of the function.